charts/agent-group/values.schema.json

```diff
@@ -0,0 +1,257 @@
+          },
+          "type": "array"
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable horizontal pod autoscaling"
+        },
+        "maxReplicas": {
+          "minimum": 1,
+          "type": "integer",
+          "description": "Maximum number of replicas"
+        }
+      }
+    },
+    "resources": {
+      "description": "Upper- and lower-bounded resource constraints of the containers",
+      "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements"
+    }
+  },
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "agent-group chart configuration",
+  "title": "Agent Group Chart Configuration",
+  "additionalProperties": false,
+  "required": [
+    "image",
+    "agent",
+    "hpa",
+    "replicas",
+    "resources",
+    "revisionHistoryLimit",
+    "affinity",
+    "env"
+  ],
+  "type": "object",
+  "properties": {
+    "replicas": {
+      "description": "Number of replicas set if the horizontal pod autoscaler is disabled or malfunctioning",
+      "$ref": "#/definitions/replicas"
+    },
+    "image": {
+      "description": "Docker image configuration",
+      "$ref": "#/definitions/image"
+    },
+    "agent": {
+      "additionalProperties": false,
+      "required": [
+        "registerjson"
+      ],
+      "type": "object",
+      "description": "Agent Group specific configuration",
+      "properties": {
+        "registerjson": {
+          "oneOf": [
+            {
+              "additionalProperties": false,
+              "type": "object"
+            },
+            {
+              "additionalProperties": false,
+              "required": [
+                "cloudUrl",
+                "username",
+                "password",
+                "agentGroupId"
+              ],
+              "type": "object",
+              "properties": {
+                "username": {
+                  "type": "string",
+                  "description": "Encrypted username of the Agent Installer user (encrypted using JitterbitUtils)"
+                },
+                "agentNamePrefix": {
+                  "type": "string",
+                  "description": "Prefix for each auto-registered agent's name"
+                },
+                "deregisterAgentOnDrainstop": {
+                  "type": "boolean",
+                  "description": "Performs agent deregistration on drainstop/JVM shutdown (defaults to false)"
+                },
+                "retryIntervalSeconds": {
+                  "minimum": 5,
+                  "type": "integer",
+                  "description": "Number of seconds it will wait before retrying. This number doubles every retry to a max of 10 minutes (defaults to 5, valid range 5-600)",
```

Comment:
"description": "Number of seconds the agent will wait before retrying. This number doubles every retry to a maximum of 600 seconds (10 minutes). Defaults to 5, valid range 5-600."